src/compound.ts

```diff
@@ -0,0 +1,73 @@
+import { ContractKit } from '@celo/contractkit'
+import { FarmBotConfig, FarmBotContract } from './types'
+import log from './log'
+import BigNumber from 'bignumber.js'
+import { AbiItem } from 'web3-utils'
+
+export function getFarmBotContract(
+  kit: ContractKit,
+  abi: AbiItem[] | AbiItem,
+  address: string,
+): FarmBotContract {
+  return new kit.web3.eth.Contract(abi, address)
+}
+
+async function getPaths({
```

Comment:
I think we'll probably eventually need some sort of enum in the `FarmBotConfig` type to represent what underlying yield farming platform is being used, since dynamic calculation of paths is probably going to depend on that. Not worth worrying about for now though.

Reply:
good point. yeah, let's iterate later